test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/007_VD_parameter_in_API_DB_app_version_less_then_parameter_version.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: The app is NOT able to retrieve the parameter in case app version is less then parameter version,
+-- parameter is listed in DB and API
+-- In case:
+-- 1) App is registered with syncMsgVersion=5.0
+-- 2) `FuelRange` parameter has since=6.0.0 in API and DB
```

Comment:
The scrip should check not FuelRange parameter but new parameters of FuelRange structure. Comment is related to scripts 007-009